core/api/make.cpp

```diff
@@ -174,13 +283,19 @@ namespace fc::api {
           }
           return result;
         }},
-        .StateGetActor = {[&](auto &address,
+        .StateGetActor = {[=](auto &address,
                               auto &tipset_key) -> outcome::result<Actor> {
           OUTCOME_TRY(context, tipsetContext(tipset_key, true));
           return context.state_tree.get(address);
         }},
+        .StateListMiners = {[=](auto &tipset_key)
+                                -> outcome::result<std::vector<Address>> {
+          OUTCOME_TRY(context, tipsetContext(tipset_key));
+          OUTCOME_TRY(power_state, context.powerState());
+          return power_state.claims.keys();
```

Comment:
```suggestion
          return power_state.getMiners();
```

Reply:
`getMiners()` filters out miners with 0 pledge while `claims.keys()` doesn't.

Reply:
No, see [the-part#yeNsEL](https://the-part.t-b-m.space/doc/yeNsEL)

Reply:
ok